### PR TITLE
refactor: placed screenreader infoTip content after button in DOM order

### PR DESCRIPTION
**Pull request by andleebsyed**
### PR Checklist

- [ ] Related to designs:
- [x] Related to JIRA ticket: [ENT-73](https://skillsoftdev.atlassian.net/jira/software/c/projects/ENT/boards/778?selectedIssue=ENT-73)
- [ ] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change
- [ ] This PR includes testing instructions tests for the code change
- [ ] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### Testing Instructions

[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](http://www.google.fr/ 'Named link title')  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](http://www.google.fr/ 'Named link title')  | [Portal Env](http://www.google.fr/ 'Named link title')   |
| Another Repo | [Another Link](http://www.google.fr/ 'Named link title') | [Another Env](http://www.google.fr/ 'Named link title')  |

[ENT-73]: https://skillsoftdev.atlassian.net/browse/ENT-73?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@55.25.4-alpha.8ed524.0<br/>@codecademy/gamut-kit@0.6.425-alpha.8ed524.0<br/>@codecademy/styleguide@66.29.5-alpha.8ed524.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://668fc7ec768e4d0c55daf523--gamut-preview.netlify.app>https://668fc7ec768e4d0c55daf523--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/668fc7ec768e4d0c55daf523>Deploy Logs</a></p>

File: packages/gamut/src/Tip/InfoTip/index.tsx [hunk 1]
```diff
@@ -81,16 +81,16 @@ export const InfoTip: React.FC<InfoTipProps> = ({
...
       <Tip {...tipProps} type="info">
         <InfoTipButton
           active={!isTipHidden}
           emphasis={emphasis}
           onClick={() => clickHandler()}
         />
       </Tip>
+      <Text screenreader aria-live="assertive">
+        {!isTipHidden ? info : ''}
+      </Text>
```
Comment (dreamwasp) [inline packages/gamut/src/Tip/InfoTip/index.tsx:93]: this InfoTip has other alignments that need to be accounted for - the DOM reading order will need to be adjusted as before the button for all `top` alignments + after the button in all `bottom` alignments
image

**Review by dreamwasp — CHANGES_REQUESTED** [review]
this needs to be a little more dynamic but then should be good! will  ✅  once the changes have been made + the testing instructions + PR envs are tested. 

since this is a Gamut accessibility issue you can also submit a request [here](https://skillsoftdev.atlassian.net/jira/software/projects/WEBREQ/boards/764) + we will prioritize getting it fixed for you

**Pull request closed without merging**